index.php

```diff
@@ -63,6 +64,16 @@
 
 // Current locks.
 echo html_writer::tag('h1', get_string('h1_current', 'tool_lockstats') . ' (' . count($records) . ')');
+$sessionkey = sesskey();
```

Comment:
this sesskey logic is wrong, the sesskey must be passed in from the outside via a get / post param or it won't actually do anything

Reply:
Using require_sesskey() which gets the sesskey from the url param